docs/Design/Classes/Class Declarations.md

```diff
@@ -6,18 +6,22 @@ This improvement adds support for class declarations to GLS, including:
 
 * Naming a class
 * Declaring any generics *(optional)*
-* Declaring a class to extend *(optional)*
+* Declaring a parent class to extend *(optional)*
+* Declaring parent interfaces to implement *(optional)*
 
 
 ## Commands
 
 ### `class start`
 
-`class start` `:` `classDescriptor`*`[, parentClassDescriptor]`*
+`class start : classDescriptor [ "extends" parentClassDescriptor] [ "implements" parentInterfaceName1 parentInterfaceName2 parentInterfaceName3 ...] `
 
 Starting a class declaration will be done with the `class start` command. 
 * The first parameter (required) will be the class' descriptor.
-* The second parameter (optional) will be a parent class' descriptor.
+* The second parameter (optional) will be the keyword "extends".
+* The third parameter (optional) will be the parent class' descriptor.
+* The fourth parameter (optional) will be the keyword "implements".
+* The fifth parameter on (optional) will be the names of interfaces to implement from.
```

Comment:
You can use phrasing like "If there are any ..." or "Optionally, ..." to break up the monotony of these bullets.